search/search_integration_tests/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ omim_link_libraries(
   stats_client
   indexer
   editor
+  editor_tests_support
```

Comment:
This should go before editor. Move it to other _tests_supports, please.

Reply:
done